src/components/Kicker/Kicker.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Kicker.scss';
+
+function Kicker ( {
+	children,
+} ) {
+	return (
+		<div className={styles.container}>
+			{children}
+		</div>
```

Comment:
Same re. nest-ability
```suggestion
		<span className={styles.container}>
			{children}
		</span>
```

Reply:
Same comment as above, and again curious about use cases.